readme.md

```diff
@@ -1,2 +1,17 @@
 # GIT Team Flow
-## We are the brodasss!! 
+## JHESS + MC + YC = <3
+## We're the the 3 best friends anyone can have!
```

Comment:
Hey Broda! I like your cool changes, but please change "friends" for "Gods".
Appreciate it